flutter/lib/resources/resource_manager.dart

```diff
@@ -23,6 +24,20 @@ class BatchPreset {
   });
 }
 
+enum ResourceTypeEnum { model, lib, datasetData, datasetGroundtruth }
```

Comment:
What is `lib` type for?
As far as I see right now it's not used anywhere.

Reply:
I have seen in the code somewhere that the app can also have a path to a lib file, so I add it, but it is not used yet, we can remove it and add it later if needed.